RELEASES.md

```diff
@@ -40,25 +46,30 @@ are backported from the `main` branch to all supported stable branches by the ma
 stable releases. New stable versions from non-critical security fixes are released on a regular
 schedule, initially aiming for the bi-weekly releases.
 
-### Release management
-
-Release managers of stable releases are responsible for approving and merging backports, tagging
-stable releases and sending announcements about them. This role is rotating on a quarterly basis.
-
-| Quarter |       Release manager                                          |
-|:-------:|:--------------------------------------------------------------:|
-| 2020 Q1 | Piotr Sikora ([PiotrSikora](https://github.com/PiotrSikora))   |
-| 2020 Q2 | Piotr Sikora ([PiotrSikora](https://github.com/PiotrSikora))   |
-| 2020 Q3 | Yuchen Dai ([lambdai](https://github.com/lambdai))             |
-| 2020 Q4 | Christoph Pakulski ([cpakulski](https://github.com/cpakulski)) |
-| 2021 Q1 | Rei Shimizu ([Shikugawa](https://github.com/Shikugawa))        |
-| 2021 Q2 | Dmitri Dolguikh ([dmitri-d](https://github.com/dmitri-d))      |
-| 2021 Q3 | Takeshi Yoneda ([mathetake](https://github.com/mathetake))     |
-| 2021 Q4 | Otto van der Schaaf ([oschaaf](https://github.com/oschaaf))    |
-| 2022 Q1 | Otto van der Schaaf ([oschaaf](https://github.com/oschaaf))    |
-| 2022 Q2 | Pradeep Rao ([pradeepcrao](https://github.com/pradeepcrao))    |
-
-## Release schedule
+### Release Management
+
+Major releases are handled by Matt Klein ([mattklein123](https://github.com/mattklein123))
+or Alyssa Wilk ([alyssawilk](https://github.com/alyssawilk)) and do not involve any backports.
```

Comment:
It would be nice if other people did this. :)

Reply:
Sure a rotation would be great. I think it might require an admin to actually make the release (I'm not sure) but that is a tiny step.